src/App/GettingStarted/getting-started/ForDevelopers/index.js

```diff
@@ -13,13 +15,13 @@ const Installation = () => (
         <h3>Include in the header</h3>
         <p>Copy-paste the following CSS code into <CodeTags type="secondary" code="<head>"/> before all the other stylesheets in order to load our CSS.</p>
         <ComponentPreview language="html" codeFigure>
-            <link rel="stylesheet" href="https://design.swedbankpay.com/v/4.7.0/styles" />
+            <link rel="stylesheet" href={`https://design.${brand}.com${basename}styles`} />
```

Comment:
Ah, this looks improved over my suggestion with the package.json way, but the actual .css link got removed it seems

Reply:
```suggestion
            <link rel="stylesheet" href={`https://design.${brand}.com${basename}styles/dg-style.css`} />
```

Reply:
🚀